### DISCUSSION TITLE
[docs] style table automations

### DISCUSSION THREAD

**Discussion by nikkimk** [RedHat-UX/red-hat-design-system#1172] [General]
### Description

Style tables are currently created manually which will be difficult to maintain.

### Suggested solution

Similar to #1003, we want to be able to automatically generate tables for the style pages of docs to make them easier to maintain.

**Comment by bennypowers** [comment]
This is different from #1034, which is about deriving css props in the cem from sources

This is more about fixing docs pages like https://github.com/RedHat-UX/red-hat-design-system/blob/fca743747e9b30818f0968487f53d83de527c5e3/elements/rh-footer/docs/10-style.md?plain=1#L39-L48

**Comment by markcaron** [comment]
Seems like this is a duplicate of the broader, more thorough discussion happening here: https://github.com/orgs/RedHat-UX/discussions/1059
